
resolution_x:   200

resolution_y:   150

control_rate:   10

joint_angle_initial_VS: [-1.38, -1.59, -1.64, 0.02, 1.17, 2.99]

epsilon:        10.0e-4

lambda:         0.03

eta:            1.5

phi_pol:        0.0

k:              20

resource_location:         "/home/cyh/Work/visual_servoing_ws/src/visual_servoing/robot_control_VS/param/"

image_polar_desired_name:  "image_polar_desired.png"

image_depth_desired_name:  "image_depth_desired.png"

camera_intrinsic: [ 605.689, 0.0, 324.855, 
                    0.0, 606.215, 236.483, 
                    0.0, 0.0, 1.0]

pose_desired: [0.124625,-0.311249,0.942121,0.261991,
              -0.953552,-0.300013,0.027022,0.199968,
              0.274238,-0.901729,-0.334182,0.776654,
              0.0, 0.0, 0.0, 1.0]

name_link0:         "base"

name_camera_frame:  "camera_polar_frame"

name_effector:      "tool0_controller"

polar_row_init:   20
polar_row_end:    60
polar_col_init:   0
polar_col_end:    0

gray_row_init:    0
gray_row_end:     0
gray_col_init:    170
gray_col_end:     80

